src/CurrentGuessCard.tsx

```diff
@@ -0,0 +1,78 @@
+import { Card, CardHeader, Divider, Flex, Text } from "@chakra-ui/react";
+import { Movie } from "./Movie-Interface";
+
+interface CurrentGuessCardProps {
+    currentGuess: Movie
+}
+
+export default function CurrentGuessCard({ currentGuess }: CurrentGuessCardProps) {
+    function formatGross() {
+        if (currentGuess.gross === "") {
+            return ""
+        }
+        return (Math.round(Number(currentGuess.gross) / 1000000) * 1000000).toLocaleString("en-US", {style:"currency", currency:"USD", maximumFractionDigits:0})
+    }
+    
+    function formatGenres() {
```

Comment:
we can combine both format genres & actors into one function & pass it an argument